### PR TITLE
[B-780] Show a back button on the send payment page

### PR DESCRIPTION
**Pull request by AnthonyBuisset**
closes B-780

### CODE REVIEW

File: frontend/src/components/Button/index.tsx [hunk 1]
```diff
@@ -59,11 +61,21 @@ export default function Button({
...
+        },
+        {
+          "px-6 py-4": size === ButtonSize.Lg && !iconOnly,
+          "p-4": size === ButtonSize.Lg && iconOnly,
+          "px-4 py-1.5": size === (ButtonSize.LgLowHeight || ButtonSize.Md) && !iconOnly,
+          "p-3.5": size === (ButtonSize.LgLowHeight || ButtonSize.Md) && iconOnly,
+          "px-4 py-2": size === ButtonSize.Sm && !iconOnly,
+          "p-2": size === ButtonSize.Sm && iconOnly,
+          "px-2 py-1": size === ButtonSize.Xs && !iconOnly,
+          "p-1": size === ButtonSize.Xs && iconOnly,
```
Comment (ofux) [inline frontend/src/components/Button/index.tsx:78]: Wouldn't it work like this? 👇 

```ts
!iconOnly && {
          "px-6 py-4": size === ButtonSize.Lg,
          "px-4 py-1.5": size === (ButtonSize.LgLowHeight || ButtonSize.Md),
          "px-4 py-2": size === ButtonSize.Sm,
          "px-2 py-1": size === ButtonSize.Xs,
},
iconOnly && {
          "p-4": size === ButtonSize.Lg,
          "p-3.5": size === (ButtonSize.LgLowHeight || ButtonSize.Md),
          "p-2": size === ButtonSize.Sm,
          "p-1": size === ButtonSize.Xs,
}
```

**Review by ofux — APPROVED** [review]

**Comment by oscarwroche** [comment]
Screenshot 2023-03-09 at 16 27 46

The font size is wrong (should be 24px) and there is insufficient padding below "Send payment"

**Comment by oscarwroche** [comment]
The font size for "All projects" should be 16px

**Review by oscarwroche — CHANGES_REQUESTED** [review]
See style comments

**Review by oscarwroche — APPROVED** [review]

**Pull request merged**